index.html

```diff
@@ -74,7 +95,19 @@ <h1>Fujifilm X/GFX/HS Firmware Patcher</h1>
 	<div class="panel" id="log"></div>
 	<div class="panel" id="info">Information on your model goes here.</div>
 	<div class="panel" id="tweaks">Tweak options will go here.</div>
+	<div id="bottom">
+		<button onclick="toggleTheme()" id="theme">Theme Toggle</button>
+	</div>
 </body>
+<script>
+	"true" == localStorage.getItem("dark") ? 
+		document.body.classList = 'dark' : 
+		document.body.classList = '';
+
+	const toggleTheme = () => {
```

Comment:
Interesting way of declaring a function...

Reply:
yea... there are like 500 ways in javascript and that's one of them.

Reply:
Would prefer the classic way, with `function`

Reply:
changed